lua/neotest-vitest/init.lua

```diff
@@ -53,7 +54,7 @@ adapter.root = function(path)
   return lib.files.match_root_pattern("package.json")(path)
 end
 
-function adapter.filter_dir(name)
+function adapter.filter_dir(name, relpath, root)
```

Comment:
Is there a reason for adding unused parameters?

Reply:
I intended to indicate to other devs that these parameters can be passed to the function, but In this modified version, `_relpath` and `_root` are marked as unused since there is a common convention in the community for that. Let me know if you prefer something else.